mlflow/pyspark/ml/_autolog.py

```diff
@@ -74,7 +74,7 @@ def get_feature_cols(
         try:
             transformer.transform(df_subset.drop(column))
         except IllegalArgumentException as iae:
-            if re.search("(.*) does not exist.", iae.desc, re.IGNORECASE):
+            if re.search("(.*) does not exist.", iae.message, re.IGNORECASE):
```

Comment:
```suggestion
            if re.search("(.*) does not exist.", str(iae), re.IGNORECASE):
```

Can we use the string representation?